src/graphics/graphics.c

```diff
@@ -53,13 +60,16 @@ void lovrGraphicsDestroy() {
 }
 
 void lovrGraphicsPresent() {
+#ifndef NO_WINDOW
   glfwSwapBuffers(state.window);
+#endif
   lovrGpuPresent();
 }
 
 void lovrGraphicsCreateWindow(int w, int h, bool fullscreen, int msaa, const char* title, const char* icon) {
-  lovrAssert(!state.window, "Window is already created");
+  lovrAssert(!state.window && !state.initialized, "Window is already created");
```

Comment:
I'm confused about this.  Wouldn't it make all calls to `lovr.graphics.createWindow` cause an error if `require('lovr.graphics')` has already been called?

Reply:
Possible. This could be related to the thing where the oculus mobile always crashes on second launch >_> i'll look into it

Reply:
state gets cleared on Destroy. this is the same as on PC, since initialized and window are both set in this function

Reply:
I see, this works because "initialized" is only set in `lovrGraphicsCreateWindow` here.